# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1296338202 18000
#      Sat Jan 29 16:56:42 2011 -0500
# Node ID c37f167a816a8dc1237d4b139f5b1669c5f9d1a4
# Parent  6212f303243ee79a070a3c70cefde24a80f88878
edit.m: As most editors open their own window, change mode to "async".

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-29  Ben Abbott  <bpabbott@mac.com>
+
+	* miscellaneous/edit.m: As most editors open their own window, change
+	mode to "async".
+
 2011-01-28  Ben Abbott  <bpabbott@mac.com>
 
 	* plot/__fltk_print__.m, plot/print.m: Fix {eps,ps,pdf}latexstandalone
 	printing for fltk, bug 32262.
 
 2011-01-28  John W. Eaton  <jwe@octave.org>
 
 	* sparse/svds.m: Use "test" instead of "testif HAVE_ARPACK".
diff --git a/scripts/miscellaneous/edit.m b/scripts/miscellaneous/edit.m
--- a/scripts/miscellaneous/edit.m
+++ b/scripts/miscellaneous/edit.m
@@ -148,19 +148,18 @@ function ret = edit (file, state)
 
   ## Pick up globals or default them.
 
   persistent FUNCTION = struct ("EDITOR", cstrcat (EDITOR (), " %s"),
                                 "HOME", fullfile (default_home, "octave"),
                                 "AUTHOR", default_user(1),
                                 "EMAIL",  [],
                                 "LICENSE",  "GPL",
-                                "MODE", "sync",
+                                "MODE", "async",
                                 "EDITINPLACE", false);
-
   ## Make sure the state variables survive "clear functions".
   mlock;
 
   if (nargin == 2)
     switch (toupper (file))
     case "EDITOR"
       FUNCTION.EDITOR = state;
     case "HOME"
